.travis.yml

```diff
@@ -1,49 +1,51 @@
 language: android
-dist: trusty
-sudo: required
+# These two lines probably don't change anything
```

Comment:
According to [the Travis docs](https://docs.travis-ci.com/user/reference/overview/), the default is Trusty with Docker, so the first line doesn't change anything while the second line does.